inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -593,13 +600,36 @@ private void highlightTextAndDisplayMessage(AjaxRequestTarget aTarget, LearningR
                 selectedLayer.getObject());
     }
 
-    private void actionRemoveHistoryItem(AjaxRequestTarget aTarget,
-            LearningRecord aRecord)
+    private void actionRemoveHistoryItem(AjaxRequestTarget aTarget, LearningRecord aRecord)
+        throws IOException, AnnotationException
     {
         aTarget.add(mainContainer);
         annotationPage.actionRefreshDocument(aTarget);
         learningRecordService.delete(aRecord);
         learningRecords.detach();
+        if (aRecord.getUserAction().equals(LearningRecordUserAction.ACCEPTED)) {
```

Comment:
It would be interesting that, before showing the confirmation dialog, the annotation editor jumped to the annotation, so that the user know what he is really going to delete.

Reply:
have added this jump in line 611